projects/golang/go/Makefile

```diff
@@ -35,8 +35,9 @@ AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output tex
 # This build is specifically focused on a debian base image for use with EKS build systems
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
 IMAGE_NAME?=golang-debian
-IMAGE_TAG?=$(GIT_TAG)-$(IMAGE_BUILD_ID)
-IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
+IMAGE_TAG?=$(GIT_TAG)-$(BUILD_ID)-$(IMAGE_BUILD_ID)
+LATEST_IMAGE=$(IMAGE_REPO)/$(IMAGE_NAME):$(GIT_TAG)
+IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG),$(LATEST_IMAGE)
```

Comment:
Will this change the versioning string for all versions (not just the debian one?). Looking at hte makefile, I'm unclear how this gets used.

Reply:
so we only use this specific makefile to build the debian images; it'll change for that only

Reply:
so moving forward, any of the EKS Go debian base images will have the new image tag schema

Reply:
this is used only as an input to `buildkit` - the `--output` arg in the image build is passed this as the name/tags of the images it should output